.github/CODEOWNERS

```diff
@@ -1,9 +1,2 @@
-*   @Datadog/rum-back @Datadog/rum-browser @Datadog/rum-mobile
-
-# Session Replay
-/samples/session-replay-schema/ @DataDog/rum-session-replay
-/schemas/session-replay/ @DataDog/rum-session-replay
-/schemas/session-replay-browser-schema.json @DataDog/rum-session-replay
-/schemas/session-replay-mobile-schema.json @DataDog/rum-session-replay
-/session-replay-browser-format.json @DataDog/rum-session-replay
-/session-replay-mobile-format.json @DataDog/rum-session-replay
+*                 @Datadog/rum-browser @Datadog/rum-mobile @Datadog/rum-back
+session-replay*   @DataDog/rum-browser @DataDog/rum-mobile @DataDog/rum-session-replay
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏about session replay files, would it make sense to separate common, mobile and browser files?
for ex, does browser team need to review session replay mobile specific changes?

Reply:
I think it makes sense, as collaboration is welcome to protect against conflicting changes, or even look for opportunities to unify changes between mobile & browser.